packages/amplify-util-mock/src/api/lambda-trigger-handler.ts

```diff
@@ -15,28 +16,42 @@ import { printer } from 'amplify-prompts';
 export const ddbLambdaTriggerHandler = async (
     context: $TSContext,
     streamArn?: string,
-    lambdaTriggerName?: string,
+    lambdaTrigger?: LambdaTrigger,
     localDynamoDBEndpoint?: Endpoint
 ): Promise<void> => {
-    if (!lambdaTriggerName) {
-        throw new Error('Name of the lambda trigger function must be specified');
+    if (!lambdaTrigger || (!lambdaTrigger?.name && !lambdaTrigger?.config)) {
```

Comment:
@phani-srikar I'm very sorry, I misread this code. It was correct earlier when these were Faults.